docs/developer-guide/parallel-execution.md

```diff
@@ -0,0 +1,21 @@
+---
+description: StreamingFast Substreams parallel Execution
+---
+
+# Parallel Execution
+
+In simple terms, parallel execution is the ability to pre-compute the execution of Substreams modules. In more detail, parallel execution is the process of a Substreams module's code executing multiple segments of blockchain data simultaneously. Substreams modules are executed in parallel, rapidly producing data for consumption in end-user applications.
+
+Parallel Execution is what's responsible for making Substreams extremely fast. StreamingFast was able to use 120 concurrent workers to completely process and store all of the ERC20/ERC721/ERC1155 token transfers on the Ethereum Mainnet in nearly 45 minutes!
```

Comment:
I'm quite unsure if those "marketing" like sentence have their place in the documentation. At least should be re-worded maybe we just say that full chain processing can be achieved in ~15 minutes (I was able in the past to perform a full chain processing in 15 minutes).

Or we eliminate it completely.

Reply:
I'm talking specifically about `StreamingFast was able to use 120 concurrent workers to completely process and store all of the ERC20/ERC721/ERC1155 token transfers on the Ethereum Mainnet in nearly 45 minutes!`. So with the first sentence, I think we should still say something about ability to fully process chain in minutes or something like that.

Reply:
Yes, this is correct. Per the style guide, we should not include any marketing in the documentation. In addition, we are **NOT** to use words like easy, fast, quick, etc.

This sentence has been removed. The update will be in the next commit.